gradle/dependencies.gradle

```diff
@@ -39,56 +39,56 @@ ext.kotlin_libs = [
     ],
     coroutines: [
         dependencies.create('org.jetbrains.kotlinx:kotlinx-coroutines-core-jvm') {
-            // https://android.googlesource.com/platform/external/kotlinx.coroutines/+/refs/tags/android-13.0.0_r32/CHANGES.md
+            // https://android.googlesource.com/platform/external/kotlinx.coroutines/+/refs/tags/android-14.0.0_r1/CHANGES.md
             version { strictly '1.6.4' }
         },
         dependencies.create('org.jetbrains.kotlinx:kotlinx-coroutines-android') {
-            // https://android.googlesource.com/platform/external/kotlinx.coroutines/+/refs/tags/android-13.0.0_r32/CHANGES.md
+            // https://android.googlesource.com/platform/external/kotlinx.coroutines/+/refs/tags/android-14.0.0_r1/CHANGES.md
             version { strictly '1.6.4' }
         },
     ],
 ]
 
 ext.std_libs = [
     androidx_core: [
-        // https://android.googlesource.com/platform/prebuilts/sdk/+/refs/tags/android-13.0.0_r32/current/androidx/Android.bp#2275
+        // https://android.googlesource.com/platform/prebuilts/sdk/+/refs/tags/android-14.0.0_r1/current/androidx/Android.bp#2420
```

Comment:
did none of these androidx libs change their versions?

Reply:
Nope. I updated the line numbers manually intentionally, to show that I went through this, and to make it easier for anyone to check.

@theimpulson noticed that they barely updated anything, we should deal with that later.